atomics/t1118/t1118.yaml

```diff
@@ -0,0 +1,19 @@
+attack_technique: T1118
+display_name: InstallUtil
+
+atomic_tests:
+- name: InstallUtil uninstall method call
+  description: |
+    Executes the Uninstall Method
+  supported_platforms:
+    - windows
+  input_arguments:
```

Comment:
see #161 - I think we need a prerequisites definition to handle the building of this dll

Reply:
Either way, but you are right this dll needs to be compiled before we run the test.